src/games/strategy/triplea/ui/MapData.java

```diff
@@ -31,7 +31,7 @@
 import games.strategy.triplea.ResourceLoader;
 import games.strategy.triplea.image.UnitImageFactory;
 import games.strategy.ui.Util;
-import games.strategy.util.PointFileReaderWriter;
+import tools.map.making.PointFileReaderWriter;
```

Comment:
It's a good catch, we should not have standard tripleA code calling into the tools packages. Some options:
- split PointFileReaderWriter up in to two,  core code vs non-core.
- move PointFileReaderWriter back ot the core code section, `games.strategy.util`